packages/create-react-admin/templates/ra-data-json-server/dataProvider.ts

```diff
@@ -0,0 +1,5 @@
+import jsonServerProvider from 'ra-data-json-server';
+
+export const dataProvider = jsonServerProvider(
+    import.meta.env.VITE_JSON_SERVER_URL
```

Comment:
I have a TS warning at this line in the generated project

```
Property 'env' does not exist on type 'ImportMeta'.
```

Reply:
Fixed